R/get_inboveg_recordings.R

```diff
@@ -84,19 +87,16 @@ get_inboveg_recordings <- function(
   if (!multiple) {
     if (missing(survey_name)) {
       survey_name <- "%"
```

Comment:
set this as a default value in the function arguments. So you don't need the if

Reply:
That's what I already suggested [here](https://github.com/inbo/inbodb/pull/35#issuecomment-872830331).  ;-)

Reply:
Thanks anyway to have a look at it, in your other suggestion I had missed the simplification myself...